seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-oracle-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/cdc/oracle/OracleCDCIT.java

```diff
@@ -221,6 +221,52 @@ public void testOracleCdcCheckDataWithNoPrimaryKey(TestContainer container) thro
                         });
     }
 
+    @TestTemplate
+    public void testOracleCdcCheckDataWithCustomPrimaryKey(TestContainer container)
```

Comment:
this e2e is passed
<img width="1469" alt="image" src="https://github.com/apache/seatunnel/assets/14371345/2af90a93-dde7-4c27-b2dd-e9d4ed928166">